packages/kbn-securitysolution-io-ts-alerting-types/src/throttle/index.ts

```diff
@@ -6,13 +6,15 @@
  * Side Public License, v 1.
  */
 
+import { TimeDuration } from '@kbn/securitysolution-io-ts-types';
 import * as t from 'io-ts';
 
-export const throttle = t.string;
+export const throttle = t.union([
+  t.literal('no_actions'),
+  t.literal('rule'),
+  TimeDuration({ allowedUnits: ['s', 'm', 'h'] }),
```

Comment:
`TimeDuration({ allowedUnits: ['h', 'd'] })`

Reply:
Updated, thanks for the catch!

For visibility: this made us realize that there is no validation for reading/updating rules, otherwise the CI would have broken. To investigate.